### ISSUE TITLE
Whacked cannot be opened on latest Win10

### ISSUE THREAD

**Issue by Tomasm21** [GitExl/WhackEd4#14]
I tried to open this program. And it failed. I'm using Microsoft Windows 10 (v2004)(10.0.19041 Build 19041)
The OS also tried to prevent to install it because there might be malicious code or virus.
I write dehacked code in text file but also want to try whacked.

![image](https://user-images.githubusercontent.com/25200026/97742333-cbe02300-1aec-11eb-958b-9c3e5cf1b2b8.png)
![image](https://user-images.githubusercontent.com/25200026/97742387-e1ede380-1aec-11eb-81db-123a367758d0.png)

**Comment by GitExl** [comment]
Does this still occur with the latest release https://github.com/GitExl/WhackEd4/releases/tag/v1.3.0-beta ?

**Comment by m21tomas** [comment]
I have installed 1.2.4. Now it is ok.
Actually when whacked has found long ago installation directory with old files and intalled on top of it then there were some files which caused the same error as above upon opening the app. I couldn't believe this happening again. So I uninstalled whacked and deleted whole directory with all files in `E:\Program Files (x86)\WhackEd4`. Then I reinstalled the 1.2.4 into `E:\Program Files\WhackEd4`. And I opened it without any errors anymore.
